Pear/Profile/InterestSettingsViewController.swift

```diff
@@ -0,0 +1,197 @@
+//
+//  AddingInterestSettingsViewController.swift
+//  Pear
+//
+//  Created by Tiffany Pan on 10/18/22.
+//  Copyright © 2022 cuappdev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class InterestSettingsViewController: UIViewController {
+
+    // MARK: - Private View Vars
+    private let backButton = UIButton()
+    private let saveButton = DynamicButton()
+    private let fadeTableView = FadeWrapperView(
+        UITableView(frame: .zero, style: .plain),
+        fadeColor: .backgroundLightGreen
+    )
+    private let titleLabel = UILabel()
+
+    // MARK: - Data
+    private weak var delegate: didUpdateInterestsDelegate?
+    private var interests: [Interest] = []
+    private var selectedInterests: [Interest] = []
+    private var updatedUser: UserV2
+    
+    init(updatingUser: UserV2, delegate: didUpdateInterestsDelegate) {
+        self.delegate = delegate
+        self.selectedInterests = updatingUser.interests
+        self.updatedUser = updatingUser
+        super.init(nibName: nil, bundle: nil)
+   }
+
+   required init?(coder: NSCoder) {
+       fatalError("init(coder:) has not been implemented")
+   }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
```

Comment:
same thing here. Create a helper function for each subview that includes adding it as a subview, configuring it, and constraining it